component/src/main/java/com/synopsys/integration/alert/component/audit/web/AuditEntryControllerLegacy.java

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
This is the old controller AuditEntryController.  The only change in this file is to rename the class.